Specs/DateTime/English/DateTimeModel.json

```diff
@@ -4610,5 +4610,130 @@
         }
       }
     ]
+  },
+  {
+    "Input": "Show sales in the year 2008",
+    "Context": {
+      "ReferenceDateTime": "2018-06-28T00:00:00"
+    },
+    "NotSupported": "java,javascript,python",
+    "Results": [
+      {
+        "Text": "year 2008",
+        "Start": 18,
+        "End": 26,
+        "TypeName": "datetimeV2.daterange",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2008",
+              "type": "daterange",
+              "start": "2008-01-01",
+              "end": "2009-01-01"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Show sales in the year",

```

Comment:
![image](https://user-images.githubusercontent.com/15103725/42148294-3ba1ee04-7e04-11e8-9712-39cad9668e3c.png)

Reply:
not sure if this is a right example